src/basic_recipes/barplot.jl

```diff
@@ -44,7 +44,7 @@ $(ATTRIBUTES)
         color_over_bar = automatic,
         label_offset = 5,
         label_font = theme(scene, :font),
-        label_size = 20,
```

Comment:
@SimonDanisch you added `bar_labels` in https://github.com/MakieOrg/Makie.jl/pull/1069. Do you remember, if there was a reason you hard-coded the font size? I think it looks nicer, if the labels have the same font size as the axis labels, tick labels etc.

Reply:
This was the old global default, but it should inherit the new value like all other functions